delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/config/BatchConfig.scala

```diff
@@ -17,7 +17,7 @@ final case class BatchConfig(maxElements: Int, maxInterval: FiniteDuration)
 
 object BatchConfig {
 
-  val individual = BatchConfig(1, 5.millis)
+  val individual = BatchConfig(1, 200.millis)
```

Comment:
Intentional?

Reply:
Yes, I have seen very low values messing things with fs2 (even if it did not help here)